packages/next/src/server/app-render.tsx

```diff
@@ -821,7 +821,7 @@ function getPreloadedFontFilesInlineLinkTags(
     return null
   }
 
-  const fontFiles = new Set<string>()
+  const fontFiles: string[] = []
```

Comment:
Why is this changed to an array? This can cause duplicates 🤔

Reply:
Oh, facepalm:.
Thanks for catching it. I changed it because it introduced non-determinism in order.
Didn't realize it's deduplicating things.

Will sort it instead :+1:.